### DISCUSSION TITLE
How is this speedup possible?

### DISCUSSION THREAD

**Discussion by alhafoudh** [openai/whisper#970] [Q&A]
Read this: https://twitter.com/JiliJeanlouis/status/1625858461933371393

**Comment by EtienneAb3d** [comment]
To get such incredible perfs, it's certainly a conversion of the original model to a much more reduced neural architecture.
Something like this?
https://arxiv.org/abs/2302.04907

**Reply by ItakeLs** [reply]
Curious about this too if anyone has the answer. Excuse my lack of knowledge but can that method be used and still maintain the same quality / WERs? According to them, it's about 60x speed-up from the original whisper.
